mq/main/mq-client/src/main/java/com/sun/messaging/jmq/jmsclient/XAJMSContextImpl.java

```diff
@@ -103,28 +104,4 @@ public JMSContext getContext() {
     public XAResource getXAResource() {
         return xaSession.getXAResource();
     }
-
-    @Override
-    public boolean getTransacted() {
-        // the API states that this should always return true
```

Comment:
:confused: Interesting thing: comments refer to _underlying XASession_ - but method from `super` is used...